ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17-periodics.yaml

```diff
@@ -17484,6 +17484,182 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build04
+  cron: 43 5 6,15,24 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-4.17
+    org: openshift
+    repo: openshift-tests-private
+  labels:
+    ci-operator.openshift.io/cloud: gcp
+    ci-operator.openshift.io/cloud-cluster-profile: gcp-qe
+    ci-operator.openshift.io/variant: amd64-nightly
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-gcp-ipi-f9-mco-techpreview-g1
```

Comment:
We need to add this "reporter_config" section to the job so that we are notified via slack when the job is executed. Like this:


```
name: periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-gcp-ipi-f9-mco-techpreview-g1
  reporter_config:
    slack:
      channel: '#forum-ocp-mco-qe'
      job_states_to_report:
      - failure
      - error
      - success
      report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow: {{else}}
        :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> :volcano: {{end}}'

```